packages/block-library/src/file/save.js

```diff
@@ -11,11 +12,31 @@ export default function save( { attributes } ) {
 		textLinkTarget,
 		showDownloadButton,
 		downloadButtonText,
+		showInlineEmbed,
+		embedHeight,
 	} = attributes;
 
 	return (
 		href && (
 			<div>
+				{ showInlineEmbed && (
+					<>
+						<object
+							className="wp-block-file__embed"
+							data={ href }
+							type="application/pdf"
+							style={ {
+								width: '100%',
+								height: `${ embedHeight }px`,
+							} }
+							aria-label={ sprintf(
+								/* translators: Placeholder is the filename. */
```

Comment:
```suggestion
								/* translators: %s: filename. */
```

Core usually uses this style for translator comments